src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -251,14 +251,35 @@ def acr_login(registry_name, resource_group_name=None, username=None, password=N
         p = Popen(["docker", "login",
                    "--username", username,
                    "--password-stdin",
-                   login_server], stdin=PIPE)
-        p.communicate(input=password.encode())
+                   login_server], stdin=PIPE, stderr=PIPE)
+        _, stderr = p.communicate(input=password.encode())
     else:
         p = Popen(["docker", "login",
                    "--username", username,
                    "--password", password,
-                   login_server])
-        p.wait()
+                   login_server], stderr=PIPE)
+        _, stderr = p.communicate()
+
+    if stderr:
+        if b'error storing credentials' in stderr and b'stub received bad data' in stderr \
```

Comment:
Is docker localized or can we take dependency on this?

Reply:
Docker CLI doesn't have localization at this point so this should be good.